### PR TITLE
update my domain

### PR DESCRIPTION
**Pull request by ajm188**

### CODE REVIEW

**Comment by mtbentley** [comment]
Needs more TLS

:+1:

**Comment by brenns10** [comment]
Wheeeeee

File: directory.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ Active Members
 
 - Tim Henderson: http://hackthology.com
 - Stephen Brennan: http://stephen-brennan.com
-- Andrew Mason: http://ajm188.github.io
+- Andrew Mason: http://fixedpoint.xyz
 - Matthew Bentley: http://bentley.link
```
Comment (ajm188) [inline directory.rst:21]: @matthewbentley you're one to talk :stuck_out_tongue:

Comment (mtbentley) [inline directory.rst:21]: `curl -v http://bentley.link`

``````
* Rebuilt URL to: http://bentley.link/
*   Trying 104.236.86.41...
* Connected to bentley.link (104.236.86.41) port 80 (#0)
> GET / HTTP/1.1
> Host: bentley.link
> User-Agent: curl/7.45.0
> Accept: */*
> 
< HTTP/1.1 301 Moved Permanently
< Server: nginx/1.8.0
< Date: Wed, 25 Nov 2015 04:07:52 GMT
< Content-Type: text/html
< Content-Length: 184
< Connection: keep-alive
< Location: https://bentley.link/
< 
<html>
<head><title>301 Moved Permanently</title></head>
<body bgcolor="white">
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx/1.8.0</center>
</body>
</html>
* Connection #0 to host bentley.link left intact```
``````

Comment (ajm188) [inline directory.rst:21]: +1. I take it back. You should update that

**Pull request merged**